tests/cucumber/features/authentication/authentication.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+// You can include npm dependencies for support files in tests/cucumber/package.json
+var _ = require('underscore');
+// You can use normal require here, cucumber is NOT run in a Meteor context (by design)
+var url = require('url');
+
+module.exports = function () {
+
+  this.When(/^I click the login button$/, function() {
+    client.url(process.env.ROOT_URL);
+
+    client.waitForExist(".container-fluid", 1000);
+    client.waitForVisible(".container-fluid", 1000);
+
+    client.click("#login-btn");
+  });
+
+  this.When(/^I enter my authentication information$/, function() {
+    return 1;
```

Comment:
Leaving Cucumber steps empty is effectively "passing". I don't think they need to return anything.